Bio/Phylo/TreeConstruction.py

```diff
@@ -545,43 +545,42 @@ class DistanceTreeConstructor(TreeConstructor):
 
     Examples
     --------
-    >>> from TreeConstruction import DistanceTreeConstructor
+    >>> from Bio.Phylo.TreeConstruction import DistanceTreeConstructor
     >>> constructor = DistanceTreeConstructor()
 
     UPGMA Tree:
 
     >>> from Bio.Phylo.TreeConstruction import DistanceCalculator
     >>> from Bio import AlignIO
-    >>> aln = AlignIO.read(open('Tests/TreeConstruction/msa.phy'), 'phylip')
+    >>> aln = AlignIO.read(open('./TreeConstruction/msa.phy'), 'phylip')
```

Comment:
Again, please just use ``'TreeConstruction/msa.phy'`` without the leading dot slash.